PrimeUnicat/solution_1/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3.11.5-alpine3.18
+
+WORKDIR /opt/app
+COPY *.cat *.py ./
+RUN pip install unicat-esolang==0.1.4
```

There's one Hadolint warning left. I think this will fix it:

```suggestion
RUN pip install --no-cache-dir unicat-esolang==0.1.4
```